apps/src/templates/feedback/CodeWritten.jsx

```diff
@@ -1,48 +1,84 @@
-var React = require('react');
-var msg = require('@cdo/locale');
+import Radium from 'radium';
+import React from 'react';
+import msg from '@cdo/locale';
 
-var CodeWritten = React.createClass({
+const styles = {
+  summary: {
+    fontSize: 18,
+    lineHeight: '20px',
+    fontWeight: 'normal',
+    outline: 'none',
+    padding: 5,
+  },
+  challengeLineCounts: {
+    fontSize: 16,
+  },
+  challengeSummary: {
+    fontColor: 'black',
+    fontSize: 14,
+    marginLeft: 40,
+  },
+  details: {
+    textAlign: 'left',
+  },
+};
+
+export default Radium(React.createClass({
 
   propTypes: {
     numLinesWritten: React.PropTypes.number.isRequired,
     totalNumLinesWritten: React.PropTypes.number.isRequired,
     children: React.PropTypes.node,
+    useChallengeStyles: React.PropTypes.bool,
   },
 
-  render: function () {
-    var lines, totalLines, showCode;
-    var styles = {
-      summary: {
-        fontSize: '18px',
-        lineHeight: '20px',
-        fontWeight: 'normal',
-        outline: 'none',
-        padding: '5px',
-      }
-    };
-
-    lines = (<p id="num-lines-of-code" className="lines-of-code-message">
+  render() {
+
+    const lines = (
+    <p
+      id="num-lines-of-code"
+      className="lines-of-code-message"
+      style={this.props.useChallengeStyles ? styles.challengeLineCounts : null}
```

Comment:
Challenge levels are meant to look significantly different from non-challenge, so this will have to be forked anyway.